exporter/prometheusremotewriteexporter/testutil_test.go

```diff
@@ -158,21 +158,23 @@ var (
 		emptyCumulativeSum:       getEmptyCumulativeSumMetric(emptyCumulativeSum),
 		emptyCumulativeHistogram: getEmptyCumulativeHistogramMetric(emptyCumulativeHistogram),
 	}
-	staleNaNIntGauge    = "staleNaNIntGauge"
-	staleNaNDoubleGauge = "staleNaNDoubleGauge"
-	staleNaNIntSum      = "staleNaNIntSum"
-	staleNaNSum         = "staleNaNSum"
-	staleNaNHistogram   = "staleNaNHistogram"
-	staleNaNSummary     = "staleNaNSummary"
+	staleNaNIntGauge       = "staleNaNIntGauge"
+	staleNaNDoubleGauge    = "staleNaNDoubleGauge"
+	staleNaNIntSum         = "staleNaNIntSum"
+	staleNaNSum            = "staleNaNSum"
+	staleNaNHistogram      = "staleNaNHistogram"
+	staleNaNEmptyHistogram = "staleNaNEmptyHistogram"
+	staleNaNSummary        = "staleNaNSummary"
 
 	// staleNaN metrics as input should have the staleness marker flag
 	staleNaNMetrics = map[string]pdata.Metric{
-		staleNaNIntGauge:    getIntGaugeMetric(staleNaNIntGauge, lbs1, intVal1, time1),
-		staleNaNDoubleGauge: getDoubleGaugeMetric(staleNaNDoubleGauge, lbs1, floatVal1, time1),
-		staleNaNIntSum:      getIntSumMetric(staleNaNIntSum, lbs1, intVal1, time1),
-		staleNaNSum:         getSumMetric(staleNaNSum, lbs1, floatVal1, time1),
-		staleNaNHistogram:   getHistogramMetric(staleNaNHistogram, lbs1, time1, floatVal2, uint64(intVal2), bounds, buckets),
-		staleNaNSummary:     getSummaryMetric(staleNaNSummary, lbs2, time2, floatVal2, uint64(intVal2), quantiles),
+		staleNaNIntGauge:       getIntGaugeMetric(staleNaNIntGauge, lbs1, intVal1, time1),
+		staleNaNDoubleGauge:    getDoubleGaugeMetric(staleNaNDoubleGauge, lbs1, floatVal1, time1),
+		staleNaNIntSum:         getIntSumMetric(staleNaNIntSum, lbs1, intVal1, time1),
+		staleNaNSum:            getSumMetric(staleNaNSum, lbs1, floatVal1, time1),
+		staleNaNHistogram:      getHistogramMetric(staleNaNHistogram, lbs1, time1, floatVal2, uint64(intVal2), bounds, buckets),
```

Comment:
I wasn't sure if I should just modify this test input to have empty bounds/buckets, if that's what a stale NaN Histogram should always look like, so I decided to adde a new case to avoid reducing coverage for that particular scenario.

Reply:
When coming from the `prometheus` receiver there will be empty bounds/buckets (at least, for now, I'm not positive it's correct), but if any other source produces metrics with that flag there may be bounds/buckets, so this is probably safest.